plugins/optimization-detective/tests/test-class-od-url-metrics-group.php

```diff
@@ -417,6 +417,84 @@ public function test_get_lcp_element( array $breakpoints, array $url_metrics, ar
 		$this->assertSame( $expected_lcp_element_xpaths, $lcp_element_xpaths_by_minimum_viewport_widths );
 	}
 
+	/**
+	 * Data provider for test_get_lcp_element_when_group_half_stale.
+	 *
+	 * @return array<string, array{order_reversed: bool}> Data.
+	 */
+	public function data_provider_test_get_lcp_element_when_group_half_stale(): array {
+		return array(
+			'original_order' => array(
+				'order_reversed' => false,
+			),
+			'reverse_order'  => array(
+				'order_reversed' => true,
+			),
+		);
+	}
+
+	/**
+	 * Test get_lcp_element() when half of the URL Metrics in a group are stale.
+	 *
+	 * @covers ::get_lcp_element
+	 * @covers OD_URL_Metric_Group_Collection::get_common_lcp_element
+	 * @dataProvider data_provider_test_get_lcp_element_when_group_half_stale
+	 *
+	 * @param bool $order_reversed Whether the order of URL Metrics should be reversed.
+	 */
+	public function test_get_lcp_element_when_group_half_stale( bool $order_reversed ): void {
+		$current_etag = 'f8527651f96776745f88cc49df70b62d';
```

Comment:
Maybe read this from any of the elements in the fixture data instead of hard-coding?

Reply:
I've addressed this in b96df7cfa by finding the ETag that is used most often as the one which is the current ETag.